docs/source/AdministratorGuide/Tutorials/index.rst

```diff
@@ -0,0 +1,16 @@
+.. _dirac-admin-tutorials:
+
+=============================
+DIRAC Administrator tutorials
+=============================
+
+Each of this tutorial is a step by step guide.
+
+.. toctree::
+   :maxdepth: 1
+   :numbered:
+
+   basicTutoSetup
+   diracSE.rst
```

Comment:
It probably works both ways, but can you either add `.rst` or not add it?

Reply:
Good point. Done